ERCS/erc-8188.md

```diff
@@ -0,0 +1,117 @@
+---
+eip: 8188
```

Comment:
```suggestion
eip: 8112
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.  You don't get to pick your own.

Please also update the filename.